### PR TITLE
Story-board-app source code added to "render-only-sample\tests\story-board-app"

### PR DESCRIPTION
**Pull request by matangpanchal**
As per the instruction of "bhouse-microsoft" , I have created storyboard demo app with Translate, scale, fade animations.

### CODE REVIEW

File: render-only-sample/tests/story-board-app/MainPage.xaml.vb [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+        Dim ctr As Int16 = 1
+
+        LayoutRoot.Resources.Add("st", entryStoryBoard)
+        LayoutRoot.Resources.Add("st1", exitStoryBoard)
+
+        While True
+            Try
+
+
+                Dim str As StorageFile = Await Windows.Storage.ApplicationData.Current.LocalFolder.GetFileAsync("IMG_0435.JPG")
```
Comment (hideyukn88) [inline render-only-sample/tests/story-board-app/MainPage.xaml.vb]: "IMG_0435.JPG" seems not provided in assets, thus causing FileNotFoundException.

Comment (matangpanchal) [inline render-only-sample/tests/story-board-app/MainPage.xaml.vb]: Hi,
Can you put any other image with the same name ? or should I add and then again do pull request ? Or can you add with the attached image ?
Please Advice ?
img_0435

Comment (matangpanchal) [inline render-only-sample/tests/story-board-app/MainPage.xaml.vb]: Hello,
I have commited changes by adding image to assets and changing code accordingly, kindly consider this for pulling. Sorry for inconvenience.

**Comment by hideyukn88** [comment]
Thank you for your contribution, I merged your pull request to master.

**Comment by matangpanchal** [comment]
Thank you very much. It's my pleasure.

**Pull request merged**